app/resources/brave_generated_resources_en-GB.xtb

```diff
@@ -308,8 +308,8 @@ By installing this extension, you are agreeing to the Google Widevine Terms of U
 <translation id="1872784258300710671">Brave uses completely private product analytics to estimate the overall usage of certain features</translation>
```

Comment:
.xtb files are generated automatically when we pull translations from Transifex. There's no need to edit them.